tests/wycheproof/test_rsa.py

```diff
@@ -0,0 +1,82 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+import binascii
+
+import pytest
+
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.backends.interfaces import RSABackend
+from cryptography.hazmat.primitives import hashes, serialization
+from cryptography.hazmat.primitives.asymmetric import padding
+
+
+_DIGESTS = {
+    "SHA-1": hashes.SHA1(),
+    "SHA-224": hashes.SHA224(),
+    "SHA-256": hashes.SHA256(),
+    "SHA-384": hashes.SHA384(),
+    "SHA-512": hashes.SHA512(),
+}
+
+
+def should_verify(backend, wycheproof):
+    if wycheproof.valid:
+        return True
+
+    if wycheproof.acceptable:
+        if (
+            backend._lib.CRYPTOGRAPHY_OPENSSL_110_OR_GREATER and
+            wycheproof.has_flag("MissingNull")
+        ):
+            return False
+        return True
+
+    return False
+
+
+@pytest.mark.requires_backend_interface(interface=RSABackend)
+@pytest.mark.supported(
+    only_if=lambda backend: (
+        not backend._lib.CRYPTOGRAPHY_OPENSSL_LESS_THAN_102
```

Comment:
Sigh, this is skipping on LibreSSL, isn't it?

Reply:
Yes, it is. These tests don't pass ATM on LibreSSL, but we'd like to run them, at least on a future patched version of Libre...

opinions @reaperhulk ?

Reply:
@alex do you have the test results somewhere online? I want to see what the failures are.

Reply:
@thaidn https://travis-ci.org/pyca/cryptography/jobs/402496876#L664; we've also independently reached out to the LibreSSL team about these.

(If you notice anything that strikes you as particularly severe, maybe let's move that conversation to email)

Reply:
(As discussed on IRC we've decided to skip on libre right now)